### ISSUE TITLE
Installation url in readme gives 404

### ISSUE THREAD

**Issue by sloorush** [fortran-lang/fpm#801]
### Description

I was going through the installation docs and found the link for `all supported installations` in the readme gives a 404.

```md
Fpm is available on many platforms and through multiple package managers, see our Documentation webpage for a list of [All Supported Installations](https://fpm.fortran-lang.org/en/how-to/installation.html).
```

This is the URL:>> https://fpm.fortran-lang.org/en/how-to/installation.html 

Even the how-to page seems empty (https://fpm.fortran-lang.org/en/how-to/index.html#how-to), but I think an issue about that should be made in the https://github.com/fortran-lang/fpm-docs repo.

### Expected Behaviour

It should redirect to a working page instead of a 404.

### Version of fpm

Not relevant

### Platform and Architecture

Not relevant

**Comment by urbanjost** [comment]
What page are you on with the broken links?  When I go to the fpm documentation the install page works but has a different URL 
 than what you are showing? Did you start off in the old documentation page, perhaps?

https://fpm.fortran-lang.org/en/install/index.html

**Comment by sloorush** [comment]
> Did you start off in the old documentation page, perhaps?

@urbanjost, I guess maybe this was the case or this has been fixed. I can't seem to find it now.

Regardless the how to documentation (https://fpm.fortran-lang.org/en/how-to/index.html) is still empty, maybe we can rename the issue to that.

**Comment by mwparis** [comment]
Hi - I believe that fixing this is just a matter of removing all instances of "/en" in the README.md. I've cloned, created a local branch, `fix_en_readme`, and tried to push but:
```
Mac: fpm $ git push --set-upstream origin fix_en_readme
ERROR: Permission to fortran-lang/fpm.git denied to mwparis.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```
Thanks in advance for advice.

**Comment by perazz** [comment]
Fixed by #959.

**Comment by mwparis** [comment]
Great. Thanks.
Any information about my question about permissions would be welcome.
